src/tokens.js

```diff
@@ -426,14 +426,118 @@ const dateTimeLiterals = {
 
 const dateTimeIdentifiers = Object.keys(dateTimeLiterals);
 
+
+/**
+ * A basic key-value store to hold context values.
+ */
+export class BaseContext {
```

Comment:
Proposal: Rename to `VariableContext`

Reply:
done: [295dd6e](https://github.com/nikku/lezer-feel/pull/18/commits/295dd6eb52536bc3809a714ade56889a56764e96)